### ISSUE TITLE
Normal forms fail simple examples

### ISSUE THREAD

**Issue by jpivarski** [bastikr/boolean.py#52]
Even if these cases are already in normal form, applying cnf/dnf shouldn't make them wrong. For boolean variables `x` and `y`, `x & y` does not have the same truth table as `x | y`.

```python
>>> alg = BooleanAlgebra()

>>> alg.cnf(alg.OR(Symbol("x"), Symbol("y")))
AND(Symbol('x'), Symbol('y'))

>>> alg.dnf(alg.AND(Symbol("x"), Symbol("y")))
OR(Symbol('x'), Symbol('y'))

>>> alg.cnf(alg.parse("a|b"))
AND(Symbol('a'), Symbol('b'))

>>> alg.dnf(alg.parse("a&b"))
OR(Symbol('a'), Symbol('b'))
```

**Comment by pombredanne** [comment]
Thanks for the report. Quite a bug.... Do you care to chip in a PR?

**Comment by jpivarski** [comment]
No PR; I didn't investigate what was wrong. There will need to be some more exhaustive unit testing on those functions, but presumably you can generate them with all combinations up to three levels deep or something. And you'd also need an automated way of verifying that the input and output are equivalent (same truth table). Presumably, there's a function like that elsewhere in the codebase.

**Comment by pombredanne** [comment]
@Kronuz @bastikr any idea to handle this ?

**Comment by bastikr** [comment]
This bug was introduced when we changed the normalize function to return a boolean expression instead of a tuple. The fix is straight-forward. Please review and merge #53 !

**Comment by pombredanne** [comment]
@bastikr since I have your attention, would you mind looking into https://github.com/bastikr/boolean.py/issues/42 so we can have a proper CI?

**Comment by pombredanne** [comment]
@jpivarski this has been merged in master.

**Comment by pombredanne** [comment]
And I released it on Pypi as v3.2 . Thank you @bastikr !
